cluster/v1alpha1/types.go

```diff
@@ -321,6 +327,46 @@ type PrioritizerConfig struct {
 	Weight int32 `json:"weight,omitempty"`
 }
 
+// Toleration represents the toleration object that can be attached to a placement.
+// The placement this Toleration is attached to tolerates any taint that matches
+// the triple <key,value,effect> using the matching operator <operator>.
+type Toleration struct {
+	// Key is the taint key that the toleration applies to. Empty means match all taint keys.
+	// If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+	// +optional
+	Key string `json:"key,omitempty"`
+	// Operator represents a key's relationship to the value.
+	// Valid operators are Exists and Equal. Defaults to Equal.
+	// Exists is equivalent to wildcard for value, so that a placement can
+	// tolerate all taints of a particular category.
+	// +optional
+	Operator TolerationOperator `json:"operator,omitempty"`
+	// Value is the taint value the toleration matches to.
+	// If the operator is Exists, the value should be empty, otherwise just a regular string.
+	// +optional
+	Value string `json:"value,omitempty"`
+	// Effect indicates the taint effect to match. Empty means match all taint effects.
+	// When specified, allowed values are NoSelect, PreferNoSelect and NoSelectIfNew.
+	// +optional
+	Effect v1.TaintEffect `json:"effect,omitempty"`
+	// TolerationSeconds represents the period of time the toleration (which must only be effective under the Evict Senario,
+	// otherwise this field should be ignored) tolerates the taint. By default,
+	// It is not set, which means tolerate the taint forever (do not evict).
```

Comment:
If it is nil, it indicates to tolerate the taint forever.

Reply:
Updated.